### PR TITLE
Add class static blocks to `preset-env`'s `shippedProposals`

### PR DESCRIPTION
**Pull request by nicolo-ribaudo**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | 
| Patch: Bug Fix?          |
| Major: Breaking Change?  |
| Minor: New Feature?      | Yes
| Tests Added + Pass?      | Yes
| Documentation PR Link    | 
| Any Dependency Changes?  |
| License                  | MIT

cc @rbuckton (just fyi) this means that Babel users can now use it with `@babel/preset-env` without needing to manually install a separate plugin.

### CODE REVIEW

File: packages/babel-preset-env/src/filter-items.js [hunk 1]
```diff
@@ -1,5 +1,10 @@
 // @flow
 
+import { lt } from "semver";
+import { minVersions } from "./available-plugins";
+
+const has = Function.call.bind(Object.hasOwnProperty);
```
Comment (nicolo-ribaudo) [inline packages/babel-preset-env/src/filter-items.js:6]:
```suggestion
// $FlowIgnore
const has = Function.call.bind(Object.hasOwnProperty);
```

**Review by hzoo — APPROVED** [review]

**Review by JLHwung — APPROVED** [review]

**Comment by JLHwung** [comment]
We should also add private brand checks to `shippedProposals`: https://chromestatus.com/feature/5006138707804160

**Review by jridgewell — APPROVED** [review]
At what stage do we usually add to shipped?

**Comment by nicolo-ribaudo** [comment]
When at least one browser ships it unflagged.

**Comment by nicolo-ribaudo** [comment]
Merged to `feat-7.14.0/preset-env`

**Pull request merged**